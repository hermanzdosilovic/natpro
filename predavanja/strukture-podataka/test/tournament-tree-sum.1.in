15 8
-2 3 7 11 8 28 -7 11 23 4 -1 -2 15 14 15
Q 0 2
U 0 9
Q 0 2
Q 5 10
Q 10 13
U 4 9
Q 2 4
Q 8 8
