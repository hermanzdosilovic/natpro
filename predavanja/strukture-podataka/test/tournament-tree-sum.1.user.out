8
19
58
26
27
23
